src/networkd.c

```diff
@@ -30,6 +30,27 @@
 #include "util.h"
 
 
+/**
+ * wowlan_triggers= string for wpa_supplicant.conf
+ */
+char *
```

Comment:
I think I would love if this was explicitly a static function as we're only using it locally, so: static char *.

Reply:
Done.